test/framework/src/main/java/org/opensearch/bootstrap/BootstrapForTesting.java

```diff
@@ -136,6 +139,18 @@ public class BootstrapForTesting {
         // Log ifconfig output before SecurityManager is installed
         IfConfig.logIfNecessary();
 
+        var sunJceProvider = java.security.Security.getProvider(SUN_JCE);
+        if (sunJceProvider != null) {
+            sunJceInsertFunction = () -> java.security.Security.insertProviderAt(
```

Comment:
Why do weed this `sunJceInsertFunction` ? It has no dependencies on bootstrap logic and could be used in tests (SecurityProviderManagerTests, SecurityProviderManagerTests directly)?